src/components/ManifestDetailSidePanel/ManifestDetailSidePanel.tsx

```diff
@@ -148,7 +219,9 @@ const ManifestDetailSidePanel: FC<ManifestDetailSidePanelProps> = ({
           </GridItem>
           <GridItem span={6}>{formatDate(lastModified)}</GridItem>
         </Grid>
-        <Button variant="tertiary">Export manifest</Button>
+        <Button variant="tertiary" onClick={exportManifest}>
+          Export manifest
+        </Button>
         <p className="manifest-details-delete-text">
```

Comment:
We should probably be using those weird [text components](https://www.patternfly.org/v4/components/text) instead of explicit p tags.  We should probably be using them for heading tags too hm....  Maybe we should go back and fix our old code too (in a separate pr), so we can get rid of some more css.

Reply:
OK I just created an issue for that so we can go back in a subsequent PR after this one.

Reply:
I think an earlier commit you had in this pr added a p tag, and I meant to comment on that one, not this unchanged line.  Oh well.  Still good to have made an issue.